examples/pumi_config.h.in

```diff
@@ -0,0 +1 @@
+#cmakedefine MFEM_USE_SIMMETRIX
```

Comment:
Is this really necessary?

Reply:
I think @cwsmith did it to make addition of SIMMETRIX clean

Reply:
My question was why is this not in `config/cmake/config.hpp.in` or similar?